src/public/orders/sales-order.ts

```diff
@@ -53,6 +57,11 @@ export interface SalesOrder extends SalesOrderIdentifierPOJO {
    */
   notes?: NotePOJO[];
 
+  /**
+   * Information regarding the original order source.
+   */
+  originalOrderSource?: OriginalOrderSource;
```

Comment:
@justin-shipstation What's an original order source?

Reply:
The concept of "original order source" is important for order sources that act as aggregators for other "sources" think BigCommerce/ChannelAdvisor. This is used in ShipStation in a few ways:
- For a seller with both ChannelAdvisor and their upstream order sources connected as Stores, we'll perform cross-store deduping using the original order source and other identifiers on the order.
- For order sources other than Amazon that send us Amazon orders, we really need to know that the customer placed the original order through Amazon, because they place special restrictions on how postage can be purchased.

Reply:
👍 Perfect, @justin-shipstation We'll want to capture this in the docs.